docs/plugin-development.rst

```diff
@@ -43,6 +43,7 @@ example will be:
     {
     "name": "My Cats Plugin",
     "description": "Allows users to manage their cats.",
+    "url": "http://girder.readthedocs.io/en/latest/plugin/mycat.html"
```

Comment:
@manthey Oh! Great catch. Thank you.
We have enough space there so I'll add proper indentation as well.